src/commands/plugins.rs

```diff
@@ -76,7 +75,7 @@ pub struct Install {
         requires("PLUGIN_NAME")
     )]
     /// Specify a particular version of the plugin to be installed from the Centralized Repository
-    pub version: Option<Version>,
+    pub version: Option<String>,
```

Comment:
Would this change not allow for invalid version strings to be passed through `clap`

Reply:
I changed this because not all plugins may do semver versioning. for example, they may do `v1.0` which the crate does not support (`v`).

Reply:
I think that should be something that can be made a requirement. thoughts?

Reply:
Sounds good. will revert